src/routes/noob.js

```diff
@@ -6,8 +6,19 @@ import * as tasks from '../database/queries/task'
 
 const router = express.Router()
 
-router.get('/', function(req, res, next){
-  res.send('noob')
+router.get('/:githubHandle', function(request, response, next){
```

Comment:
you don't need the `:githubHandle` query param if you are getting the github handle from the current user.